### PR TITLE
data-test id added in pending admin part

### PR DESCRIPTION
**Pull request by Ajay-aot**
JIRA:
Issue Type: BUG/ FEATURE

https://aottech.atlassian.net/browse/FWF-2909
data-test id added in pending admin part

### CODE REVIEW

File: forms-flow-admin/src/components/roles/roles.tsx [hunk 1]
```diff
@@ -547,12 +548,13 @@ const Roles = React.memo((props: any) => {
...
                   setRoles(props.roles);
                 }}
+                data-testid="clear-role-search-button"
               >
                 {t("Clear")}
               </Button>
             )}
           </div>
-          <Button variant="primary"  onClick={handleShowRoleModal}>
+          <Button variant="primary"  onClick={handleShowRoleModal} data-testid="roles-creat-new-role-button">
```
Comment (fahad-aot) [inline forms-flow-admin/src/components/roles/roles.tsx:557]: "create" spell check

**Review by fahad-aot — COMMENTED** [review]
Can you check and confirm that data-testid for elements inside create role modal and edit role modal is given?

File: forms-flow-admin/src/components/roles/roles.tsx (hunk 1 above)
Comment (Ajay-aot) [reply forms-flow-admin/src/components/roles/roles.tsx]: updated

File: forms-flow-admin/src/components/roles/roles.tsx [hunk 2]
```diff
@@ -358,14 +359,15 @@ const Roles = React.memo((props: any) => {
         </Modal.Body>
         <Modal.Footer>
           <button type="button"
-            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal}>
+            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal} data-testid="create-role-modal-edit-cancel-btn">
```
Comment (shuhaib-aot) [inline forms-flow-admin/src/components/roles/roles.tsx:362]: create-role-modal-edit-cancel-btn this is for create or edit. little bit confusing

File: forms-flow-admin/src/components/users/users.tsx [hunk 3]
```diff
@@ -352,12 +354,16 @@ const Users = React.memo((props: any) => {
...
           <div className="user-filter-container  col-lg-4 col-xl-4 col-md-4 col-sm-6 col-12 d-flex justify-content-end gap-2">
             <span className="my-2">{t("Filter By:")} </span>
-            <Form.Select className="bg-light text-dark" onChange={handleSelectFilter} title={t("Filter here")}>
-              <option value="ALL" selected={!props.filter}>
+            <Form.Select 
+             className="bg-light text-dark"
+             onChange={handleSelectFilter} 
+             title={t("Filter here")} 
+             data-testid="users-roles-filter-select">
+              <option value="ALL" selected={!props.filter} data-testid="users-all-roles-option">
```
Comment (shuhaib-aot) [inline forms-flow-admin/src/components/users/users.tsx:362]: the options need an uniformity in data testid 
example: `users-roles-filter-option-all ` for all users
                `users-roles-filter-option-${variable}` for other option

File: forms-flow-admin/src/components/roles/roles.tsx (hunk 2 above)
Comment (Ajay-aot) [reply forms-flow-admin/src/components/roles/roles.tsx]: changed

File: forms-flow-admin/src/components/users/users.tsx (hunk 3 above)
Comment (Ajay-aot) [reply forms-flow-admin/src/components/users/users.tsx]: updated

**Review by fahad-aot — APPROVED** [review]

File: forms-flow-admin/src/components/roles/roles.tsx [hunk 4]
```diff
@@ -358,14 +359,15 @@ const Roles = React.memo((props: any) => {
         </Modal.Body>
         <Modal.Footer>
           <button type="button"
-            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal}>
+            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal}  data-testid="edit-role-modal-cancel-button">
```
Comment (shuhaib-aot) [inline forms-flow-admin/src/components/roles/roles.tsx:362]: here you used button but above used btn end of the data testid so make them similar

Comment (shuhaib-aot) [reply forms-flow-admin/src/components/roles/roles.tsx:362]: `button `is you used more

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

File: forms-flow-admin/src/components/roles/roles.tsx (hunk 4 above)
Comment (Ajay-aot) [reply forms-flow-admin/src/components/roles/roles.tsx:362]: updated

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**